contrib/tracefile-analyser.php

```diff
@@ -15,7 +16,20 @@
    +----------------------------------------------------------------------+
    | Authors: Derick Rethans <derick@xdebug.org>                          |
    +----------------------------------------------------------------------+
- */
+*/
+
+
+/*
+   php.ini additions for operation:
+      xdebug.profiler_enable = 1
+      debug.trace_format = 1
+      xdebug.collect_includes = 1   (optional)
```

Comment:
I don't believe this is useful, as the analyser only reports function names, and this is for showing arguments to functions.